### DISCUSSION TITLE
Possible to test multiple apps / testslots?

### DISCUSSION THREAD

**Discussion by lehmannk** [fzyzcjy/flutter_convenient_test#353] [Q&A] [answered]
I have a use case where I need to test 2 apps interleaving a workflow at the same time. Now, `convenientTestMain` requires a `ConvenientTestSlot` whose name assumes some kind of interchangeability. Is that possible at all? `convenientTestMain` currently comments with "Please make this the only method in your "main" method."

**Comment by fzyzcjy** [comment, marked as answer]
Hi, this does not seem to be possible directly, but there may be ways to do it: It seems that two apps are isolated (e.g. two processes), so the code in one app cannot execute the code in another app *directly*. However, if you manage to make them communicate (e.g. via interprocess communication, a dummy web server, or whatever), you can drive another app from the (test) code of one app.

**Reply by lehmannk** [reply]
ok, thank you very much for the quick reply

**Reply by fzyzcjy** [reply]
You are welcome!
